index.d.ts

```diff
@@ -435,7 +474,8 @@ export type ExecaReturnValue<StdoutStderrType extends StdoutStderrAll = string>
 
 	This is `undefined` if either:
 	- the `all` option is `false` (default value)
-	- `execaSync()` was used
+	- the synchronous methods are used
+  - both `stdout` and `stderr` options are set to [`'inherit'`, `'ipc'`, `'ignore'`, `Stream` or `integer`](https://nodejs.org/dist/latest-v6.x/docs/api/child_process.html#child_process_options_stdio)
```

Comment:
```suggestion
	- both `stdout` and `stderr` options are set to [`'inherit'`, `'ipc'`, `'ignore'`, `Stream` or `integer`](https://nodejs.org/dist/latest-v6.x/docs/api/child_process.html#child_process_options_stdio)
```

Reply:
It appears that no using indentation shows like this with VSCode:

![Screenshot from 2023-12-20 20-30-26](https://github.com/sindresorhus/execa/assets/8136211/266a9026-11f5-470d-a157-fe55c9d1dfae)

But using indentation shows like this:

![Screenshot from 2023-12-20 20-30-44](https://github.com/sindresorhus/execa/assets/8136211/7570e077-e47f-4f19-851e-d3299a59088d)

However, we were being inconsistent with both the types and the `readme.md`, so I fixed it all to make it consistent.